### ISSUE TITLE
org.apache.logging.log4j.Logger.atWarn not working 

### ISSUE THREAD

**Issue by rhierlmeier** [ops4j/org.ops4j.pax.logging#412]
I tried to use Apache Poi 5.1.0 und Karaf 4.3.3. 
It is not working. I got the following Exception:

`
Caused by: java.lang.NoClassDefFoundError: org/apache/logging/log4j/internal/DefaultLogBuilder
at org.apache.logging.log4j.spi.AbstractLogger$LocalLogBuilder.initialValue(AbstractLogger.java:2953)
at org.apache.logging.log4j.spi.AbstractLogger$LocalLogBuilder.initialValue(AbstractLogger.java:2945)
at java.lang.ThreadLocal.setInitialValue(ThreadLocal.java:180)
at java.lang.ThreadLocal.get(ThreadLocal.java:170)
at org.apache.logging.log4j.spi.AbstractLogger.getLogBuilder(AbstractLogger.java:2930)
at org.apache.logging.log4j.spi.AbstractLogger.atLevel(AbstractLogger.java:2923)
at org.apache.logging.log4j.spi.AbstractLogger.atWarn(AbstractLogger.java:2882)
at org.apache.poi.util.XMLHelper.logThrowable(XMLHelper.java:307)
at org.apache.poi.util.XMLHelper.trySet(XMLHelper.java:287)
at org.apache.poi.util.XMLHelper.getDocumentBuilderFactory(XMLHelper.java:114)
at org.apache.poi.util.XMLHelper.<clinit>(XMLHelper.java:85)
at org.apache.poi.ooxml.util.DocumentHelper.newDocumentBuilder(DocumentHelper.java:47)
at org.apache.poi.ooxml.util.DocumentHelper.<clinit>(DocumentHelper.java:36)
at org.apache.poi.openxml4j.opc.internal.ContentTypeManager.save(ContentTypeManager.java:429)
at org.apache.poi.openxml4j.opc.ZipPackage.saveImpl(ZipPackage.java:554)
at org.apache.poi.openxml4j.opc.OPCPackage.save(OPCPackage.java:1487)
at org.apache.poi.ooxml.POIXMLDocument.write(POIXMLDocument.java:227)
at org.apache.poi.xssf.streaming.SXSSFWorkbook.write(SXSSFWorkbook.java:963)
Caused by: java.lang.ClassNotFoundException: org.apache.logging.log4j.internal.DefaultLogBuilder not found by org.ops4j.pax.logging.pax-logging-api [7]
at org.apache.felix.framework.BundleWiringImpl.findClassOrResourceByDelegation(BundleWiringImpl.java:1565)
at org.apache.felix.framework.BundleWiringImpl.access$300(BundleWiringImpl.java:78)
at org.apache.felix.framework.BundleWiringImpl$BundleClassLoader.loadClass(BundleWiringImpl.java:1950)
at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
... 32 more
`

It seems that it is not possible to use the `org.apache.logging.log4j.spi.AbstractLogger.atLevel` method in pax-logging

**Comment by jbonofre** [comment]
Do you use the POI SMX bundle ?

**Comment by rhierlmeier** [comment]
I need Apache POI 5.1.0. The last POI SMX Bundle is 4.1.2_2

**Comment by grgrzybek** [comment]
While I've carefully decided which packages from several Log4j2 jars should be split across pax-logging-api and pax-logging-log4j2, `org/apache/logging/log4j/internal` package was never exported/imported/private-packaged from neither of these bundles.

I decided to private-package them in pax-logging-api (after all it's `internal`). But I've added an integration test that initially failed, and after private-packaging it started to work.
